README.md

```diff
@@ -1,98 +1,163 @@
-# Alfresco in Docker
+# Alfresco in Docker and Alfresco Share in Docker
 
 > [![Xenit Logo](https://xenit.eu/wp-content/uploads/2017/09/XeniT_Website_Logo.png)](https://xenit.eu/open-source)
 >
 > docker-alfresco is a part of the Xenit Open Source Tooling around Alfresco. Xenit is company with a big expertise and
-> a strong team around Alfresco. If you'd like to learn more about our 
-> [tools](https://xenit.eu/open-source), [services](https://xenit.eu/alfresco) and 
+> a strong team around Alfresco. If you'd like to learn more about our
+> [tools](https://xenit.eu/open-source), [services](https://xenit.eu/alfresco) and
 > [products](https://xenit.eu/alfresco-products), please visit our [website](https://xenit.eu).
 
-This project builds Alfresco-specific docker images used by Xenit, starting with Alfresco 4.2.
+This project builds Alfresco-specific docker images used by Xenit, starting with Alfresco 6.1.
 
-## Images created 
+## Images created
```

Comment:
please check and correct the full image url's. Now it is inconsistent between share and repository